src/browser/documentation/help/drop-database.jsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2002-2019 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react'
+import ManualLink from 'browser-components/ManualLink'
+import DbsOnSystemDb from './partials/dbs-on-systemdb'
+const title = 'DROP DATABASE'
+const subtitle = 'Delete a database'
+const category = 'administration'
+const content = (
+  <>
+    <p>
+      The <code>DROP DATABASE</code> command can be used to delete a database.
```

Comment:
this is worded slightly different to the others, but I think is probably better worded in this way and in line with the rest of the help files.

Also, some places use `clause` and others `command` so maybe we need to be more exact with the wording with regards to each?

Sorry to be pedantic here; I feel like this is best answered by someone with better understanding of our wording/expectations from the help/docs.

Reply:
This is the most helpful answer I found for the difference between `clause` and `command`. It's in the context of SQL, but I feel like it should have the same meaning in other query languages. 

https://stackoverflow.com/questions/52835714/what-is-difference-between-clause-command-statement-and-query-in-sql

Reply:
Yeah I was looking at the same thing. I feel someone more knowledgeable with the docs and the terminology should be taking a look over this as we want to keep the wording accurate and consistent across all the Neo4j products.

Reply:
I think it sounds like a good idea to bring in someone from docs to review the wording we use. Could you ping them and ask for some or their time @nglgzz ?

Reply:
thanks @oskarhane, @jk05  & @nglgzz.

We're a little inconsistent with this, even if it's just about our definition and usage:

- **Clause** is very much used in the Cypher manual, as per https://neo4j.com/docs/cypher-manual/4.0/clauses/ 

- the newer multi-database administration section clearly prefers **command**: https://neo4j.com/docs/cypher-manual/4.0/administration/databases/#administration-databases-drop-database

I think this is a good candidate to discuss with the Terminology group, and publish our definition (both internally via a style & terminology guide, and externally as a description in the documentation)